packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
Depressingly, there really is no better way to do this. I suppose this is fine then...